app/state.go

```diff
@@ -1253,6 +1271,10 @@ func (ue *updateErrorState) Update() *datastore.UpdateInfo {
 	return &ue.update
 }
 
+func (ue *updateErrorState) PermitLooping() bool {
```

Comment:
Q:
Why is this needed, if the default implementation is `false` ? :thinking:

Reply:
Unlike most of the update states, `updateErrorState` doesn't have `updateState` as a base component. I looked into changing this, but this turned out to be quite tricky because it has `errorState` as a base component instead, so if you try to use both you get ambiguous member functions. Therefore we need to provide it manually here to fulfill the interface.

You can see that it already provides `Update()`, which is there for the same reason.